contrib/builder/deb/ppc64le/ubuntu-xenial/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM ppc64le/ubuntu:xenial
 
 RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git golang-go libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev libsqlite3-dev libsystemd-dev pkg-config --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV GO_VERSION 1.6.3
+ENV GO_VERSION 1.7.1
 ENV GO_DOWNLOAD_URL https://golang.org/dl/go${GO_VERSION}.src.tar.gz
```

Comment:
Actually this is intentional. Golang doesn't have ppc64le binaries so we have to pull from here to build from source. See the main ppc64le dockerfile as well https://github.com/docker/docker/blob/master/Dockerfile.ppc64le#L91. I'll go ahead an add in a comment here.